bundle/EventDispatcher/SyliusEventDispatcher.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\LayoutsSyliusBundle\EventDispatcher;
+
+use Sylius\Bundle\ResourceBundle\Controller\EventDispatcherInterface;
+use Sylius\Bundle\ResourceBundle\Controller\RequestConfiguration;
+use Sylius\Bundle\ResourceBundle\Event\ResourceControllerEvent;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface as SymfonyEventDispatcherInterface;
+
+use function sprintf;
+
+final class SyliusEventDispatcher implements EventDispatcherInterface
+{
+    public function __construct(
+        private readonly EventDispatcherInterface $innerEventDispatcher,
```

Comment:
Please do not use `readonly`.

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/14/commits/56da71277ab6a939078ba59b35029049625e9d4b